cmake/OpenCVDetectPython.cmake

```diff
@@ -177,7 +177,7 @@ if(NOT ${found})
 
     if(NOT ANDROID AND NOT IOS)
       if(CMAKE_HOST_UNIX)
-        execute_process(COMMAND ${_executable} -c "from distutils.sysconfig import *; print(get_python_lib())"
+        execute_process(COMMAND ${_executable} -c "from sysconfig import *; print(get_python_lib())"
```

Comment:
Looks like this doesn't work as expected (Linux x64):
```
-- Found PythonInterp: /usr/bin/python2.7 (found suitable version "2.7.6", minimum required is "2.7") 
-- Found PythonLibs: /usr/lib/x86_64-linux-gnu/libpython2.7.so (found suitable exact version "2.7.6") 
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'get_python_lib' is not defined
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.4.3", minimum required is "3.2") 
-- Found PythonLibs: /usr/lib/x86_64-linux-gnu/libpython3.4m.so (found suitable exact version "3.4.3") 
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'get_python_lib' is not defined
```

Reply:
I'm sorry, I didn't change the function here. Fixed it.